### DISCUSSION TITLE
user-guide/misc/wps-plugin-vba

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#237] [文档评论区]
# user-guide/misc/wps-plugin-vba

Zotero 非官方中文维护小组

https://zotero-chinese.com/user-guide/misc/wps-plugin-vba

**Comment by TaoChunxing** [comment]
请问为什么按照上述步骤装了VBA之后点击插件插入文献还是什么反应都没有呢？

**Reply by jiangfeiniao** [reply]
必须打开zotero

**Reply by TaoChunxing** [reply]
Zotero本是运行状态，我会使用Zotero. 是VBA版本与WPS版本不匹配导致问题未解决，后面安装了其他版本的vba之后zotero就能正常使用了

**Reply by BillZhucn** [reply]
烦请问哪里能找到wps的更新版本的vba啊

**Comment by sincerely-hello-world** [comment]
赞👍，好详细的解决方案。 感谢记录并分享。

**Comment by jiangfeiniao** [comment]
大佬啊 请接收文科生的膝盖 ！！！

**Comment by jiangfeiniao** [comment]
这是一封自动回复的邮件。  您好！  来函妥收，请君勿念，择日详复，还望见谅。                                                                祝好！                                                                穆静

**Comment by jiangfeiniao** [comment]
这是一封自动回复的邮件。  您好！  来函妥收，请君勿念，择日详复，还望见谅。                                                                祝好！                                                                穆静

**Comment by jiangfeiniao** [comment]
不知道啊 我记得自己随便找了找 好像就行



发自我的iPhone


------------------ Original ------------------
From: Bill ***@***.***&gt;
Date: Wed,Apr 22,2026 4:17 PM
To: zotero-chinese/wiki ***@***.***&gt;
Cc: jiangfeiniao ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [zotero-chinese/wiki] user-guide/misc/wps-plugin-vba (Discussion #237)





 
烦请问哪里能找到wps的更新版本的vba啊
